### PR TITLE
docs: Add guide - "how to star a repo"

### LINKED ISSUE #66: [DOCS] Add process/guide - "How to star a repo"
### Changes/Action required
 
 - Add a complete process of **"How to star a repo"**, steps will contain a text, screenshot, screen recording, and GIF (if needed)
 
- Path of the file in which **steps** need to be added - [`open-source-with-pradumna/pages/How-to/guide/star-repo.md`](https://github.com/Pradumnasaraf/open-source-with-pradumna/blob/main/pages/How-to/guide/star-repo.md)

---

If you have any suggestions feel free to Open an [Issue](https://github.com/Pradumnasaraf/open-source-with-pradumna/issues)

#### This addition of the documentation will get added and hyperlinked in [`How-to/README.md`](https://github.com/Pradumnasaraf/open-source-with-pradumna/tree/main/pages/How-to) and also get hosted on the website https://Opensource.pradumnasaraf.co

Screenshot from 2022-05-15 14-11-21

### PR DESCRIPTION
**Pull request by code-hyker**
Closes #66

👨‍💻 Changes proposed

Added the complete procedure for starring a repository along with the necessary screenshots

✔️ Check List (Check all the applicable boxes)

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by Pradumnasaraf** [comment]
Hey,
Thank you for opening the PR and for your effort.

There are some changes required.

- bolding on the second line doesn't work 
Screenshot 2022-06-17 at 10 44 33 PM

- Can you make the full or half screenshot, so that beginners can get more clarity. (Suggestion: you can point with a arrow in that screenshot)

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,
Thank you for opening the PR and for your effort.

There are some changes required.

- bolding on the second line doesn't work 
Screenshot 2022-06-17 at 10 44 33 PM

- Can you make the full or half screenshot, so that beginners can get more clarity. (Suggestion: you can point with an arrow in that screenshot)



__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by code-hyker** [comment]
The changes have been made.

File: pages/How-to/guide/star-repo.md [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+### Steps to star a repository :
+
+1. The process of starring a repository is quite simple and straightforward.Firstly,navigate to the main page of the repository.On the main page,click on the `Star` button located at the top right corner.
+
+
 <div align="center">
-  <img src="https://user-images.githubusercontent.com/74750414/169188784-680014eb-ae5a-4062-b9c0-40276f28a261.png" />
+  <img src="https://user-images.githubusercontent.com/102421475/174355850-e6a746da-e8ca-4296-9722-d87c7e3d5a92.jpg">
 </div>
+<br><br><br>
```
Comment (Pradumnasaraf) [inline pages/How-to/guide/star-repo.md:13]:
```suggestion

<br><br>
```

File: pages/How-to/guide/star-repo.md [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+  
+ 
+2. Once you have clicked on the `Star` button,it will change from `Star` to `Starred` which indicates that the repository has been starred.
+ 
+ 
+<div align="center">
+  <img src="https://user-images.githubusercontent.com/102421475/174355896-850c6c51-ba8f-4d9a-b860-ef8dbea39dcb.jpg">
+</div>
+<br><br><br>
```
Comment (Pradumnasaraf) [inline pages/How-to/guide/star-repo.md:22]:
```suggestion

<br><br>
```

File: pages/How-to/guide/star-repo.md [hunk 3]
```diff
@@ -1,3 +1,36 @@
...
+
+
+3. Once the above steps are done,you can find the starred repository by navigating to the `Stars` page as it is now present in the list of `Starred` repositories.
+
+
+<div align="center">
+  <img src="https://user-images.githubusercontent.com/102421475/174355944-10cfd324-45e3-4403-bb7d-d0edd2b2f55b.jpg">
+</div>
+<br><br><br>
```
Comment (Pradumnasaraf) [inline pages/How-to/guide/star-repo.md:31]:
```suggestion

<br><br>
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Thank you for the changes, still, some elements are not working on the hosted onw, looks like there is a problem another problem. To temp fix that these changes are necessary. You can see in the image below


Screenshot 2022-06-18 at 12 12 18 AM

**Comment by code-hyker** [comment]
I am unable to figure out why is it not working on the hosted one.

**Comment by Pradumnasaraf** [comment]
> I am unable to figure out why is it not working on the hosted one.

Likewise. Only that particular line is not working. No worries!!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Reusable

**Comment by code-hyker** [comment]
My pleasure!

**Pull request merged**